synse/cache.py

```diff
@@ -480,6 +480,9 @@ def _build_scan_cache(metainfo):
             ref['rack']['boards'] = list(ref['boards'].values())
             scan_cache['racks'].append(ref['rack'])
 
+        # Sort the scan cache by racks['id']
+        scan_cache['racks'] = sorted(scan_cache['racks'], key=lambda rck: rck['id'])
```

Comment:
this is okay, but I made this note in another PR - I'm not sure I like ordering things here. we did that in 1.4 for some stuff, and then other pieces were built off the assumption of that order. in some cases that ended up being not great.

I think most consumers of the API will be programmatic usage, so the order shouldn't matter (can be sorted as seen fit by the client). 

I think this is okay here for now, but I'd really like to avoid explicitly ordering things moving forward.